src/components/CreateWallet.tsx

```diff
@@ -84,7 +84,13 @@ export class WalletForm extends React.Component<Props> {
   }
 
   checkValidPassword = () => {
-    if (this.props.password !== this.props.passwordVerify) {
+    const password = this.props.password
+    const lengthOfPassword = password.length
+    if (lengthOfPassword < 12 || lengthOfPassword > 30) {
+      throw new InputError('Password needs to be between 12 and 30 characters long', `input-${kebabCase('password')}`)
+    }
+
+    if (password !== this.props.passwordVerify) {
```

Comment:
seeing we're already accessing this.props to get a copy of the password, it'd be nice to also grab the passwordVerify while we're doing that, and we can abbreviate it to just this.props eg:

```js
const { password, passwordVerify } = this.props
/* ... as above ... */
```

Reply:
done